<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-2D35UU5" failures="1" tests="2" name="Default test(failed)" time="0.079" errors="0" timestamp="11 May 2020 21:19:03 GMT">
  <testcase classname="demo.TestNGRetryFailedDemo" name="test2" time="0.012"/>
  <testcase classname="demo.TestNGRetryFailedDemo" name="test3" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
